### DISCUSSION TITLE
Recreating shape modification from https://opentype.js.org/ landing page

### DISCUSSION THREAD

**Discussion by Sigurdmazanti** [opentypejs/opentype.js#711] [Q&A]
Hello,

I stumbled upon [opentypeJS](https://opentype.js.org/) today and fell in love with the morphing-effect that is created when I was changing the "strength", "distance", "x" and "y"-values in one of the canvases.

E.g.:

1. 0% strength: ![0% strength set](https://i.gyazo.com/03d5ea952b71c1a414611760d62d5d7b.png)
3. 50% strength: ![50% strength set](https://i.gyazo.com/cada4ca2a8d4501d793a4a84d5603101.png)
4. 100% strength: ![100% strength set](https://i.gyazo.com/97059499719a67605df6a1e7e5e449da.png)

I'd like to recreate this in a similiar way, however not with any controls.
**Ideal scenario would be**
1. Drawing the text in the canvas with `direction=33%` and `strength=0`
2. In an interval of every 50ms, add 1% to the existing `strength`-value, until `strength=92` 

I would greatly appreciate any insights, guidance, or resources you can provide to help me understand and implement this morphing effect.

Thank you in advance.

**Comment by Connum** [comment]
I'm closing this as it's not directly related to our library. You can take a look at the code of the demo page to see how the path is being morphed, and take a look at an animation library like [GSAP](https://gsap.com/)
